src/components/networks/createNetworkDialog.jsx

```diff
@@ -53,37 +53,49 @@ function validateParams(dialogValues) {
         validationFailed.name = _("Name should not be empty");
 
     if (dialogValues.ip === "IPv4 only" || dialogValues.ip === "IPv4 and IPv6") {
+        let ipv4_prefix = null;
+
         if (isEmpty(dialogValues.netmask.trim()))
             validationFailed.netmask = _("Mask or prefix length should not be empty");
-        else if (!utils.validateNetmask(dialogValues.netmask))
-            validationFailed.netmask = _("Invalid IPv4 mask or prefix length");
+        else {
+            ipv4_prefix = utils.parseNetmask(dialogValues.netmask);
+            if (ipv4_prefix === null)
+                validationFailed.netmask = _("Invalid IPv4 mask or prefix length");
+            else if ((ipv4_prefix % 8) != 0) {
+                // because we use localPtr="yes".
+                validationFailed.netmask = _("IPv4 prefix length must be a multiple of 8");
+            } else if (ipv4_prefix == 32) {
+                // because we use localPtr="yes".
+                validationFailed.netmask = _("IPv4 prefix length must be 24 or less");
```

Comment:
TIL! So far I thought that e.g. /28 networks would be a thing.

However, this looks inconsistent: You check for `== 32` but say "24 or less" -- should that be `> 24` then?

Reply:
Yes, it's complicated.

 /28 subnets are totally a thing, but libvirt is not able to handle them with localPtr="yes", which we set in libvirt-xml-create.js in the network XML. I don't know if that is a fundamental problem, or just a missing feature in libvirt. I don't even know what localPtr="yes" does.

In fact, the only valid values are 8, 16, and 24. We can as well have a dropdown for them.

I'll change "== 32" check as you suggest.